syne_tune/try_import.py

```diff
@@ -38,6 +38,12 @@ def try_import_raytune_message() -> str:
         tag="raytune"
     )
 
+def try_import_backend_message(backend_type: str) -> str:
+    return (
+        f"{backend_type} is not imported"
+        + " since dependencies are missing. You can install them with\n"
+        + "   pip install 'syne-tune[extra]'"
+    )
```

Comment:
Is there a reason for not prefering the same convention as in methods above?
```suggestion
    return _try_import_message(
        f"Backend {backend} not imported",
        tag="extra",
    )
```

Reply:
The import for backend would not necessarily require specific tag, so in purpose of providing a better context, I've created separate function. Do you have any other suggestion on this part?

Reply:
No I see now why you had to create it, lets keep it this way.